sire/advanced.md

```diff
@@ -36,60 +36,122 @@ First parameter is a bar ("needle"), second parameter is a bar ("haystack"), thi
 
 ## Col macro
```

Comment:
It feels like this section would benefit from more of a framing instead of jumping straight into an example?

Reply:
How do you feel about line 39?

Reply:
I like it!